lib/rucio/daemons/bb8/common.py

```diff
@@ -748,36 +729,26 @@ def get_active_locks(
     session: Optional[Session] = None
 ) -> dict[str, dict[str, int]]:
     locks_dict: dict[str, dict[str, int]] = {}
-    rule_ids = (
-        session.query(models.ReplicationRule.id)
-        .filter(
-            or_(
-                models.ReplicationRule.state == RuleState.REPLICATING,
-                models.ReplicationRule.state == RuleState.STUCK,
-            ),
-            models.ReplicationRule.comments == "Background rebalancing",
-        )
-        .all()
+    stmt = select(
+        models.ReplicationRule.id
+    ).where(
+        and_(or_(models.ReplicationRule.state == RuleState.REPLICATING,
+                 models.ReplicationRule.state == RuleState.STUCK),
+             models.ReplicationRule.comments == "Background rebalancing")
     )
-    for row in rule_ids:
-        rule_id = row[0]
-        query = (
-            session.query(
-                func.count(),
-                func.sum(models.ReplicaLock.bytes),
-                models.ReplicaLock.state,
-                models.ReplicaLock.rse_id,
-            )
-            .filter(
-                and_(
-                    models.ReplicaLock.rule_id == rule_id,
-                    models.ReplicaLock.state != LockState.OK,
-                )
-            )
-            .group_by(models.ReplicaLock.state, models.ReplicaLock.rse_id)
```

Comment:
This `group_by` statement is missing in the new query

Reply:
Re-added this